src/release.js

```diff
@@ -189,16 +232,29 @@ module.exports = function taoExtensionReleaseFactory(baseBranch, branchPrefix, o
         },
 
         /**
-         * Check if release exists
+         * Gets the branch with highest version
+         * @private
```

Comment:
Actually we kept it in the module exports to facilitate unit testing (of that method and the other which calls it). If it's a better practice, I can move the 2 `@private`-tagged methods upstairs, but I don't see what benefits we would gain.

Reply:
This depends more on the API design you want to achieve. You have the 2 choices : 
1. you can expose the method, as it is (without the `@private` annotation) even if you don't actually use it (for now). But if the method can be used by itself and serves a purpose, you can expose it.
2. the method is clearly an internal thing you don't want someone to call it because it doesn't make sense in the context of your API, then move it as a module function or hidden in the factory scope.